packages/react-native-web/src/exports/Image/__tests__/index-test.js

```diff
@@ -106,10 +116,6 @@ describe('components/Image', () => {
 
   describe('prop "onLoad"', () => {
     test('is called after image is loaded from network', () => {
-      jest.useFakeTimers();
-      ImageLoader.load = jest.fn().mockImplementation((_, onLoad, onError) => {
-        onLoad();
-      });
```

Comment:
Fake timers no longer needed to pass this test

Reply:
Actually the fake timers are breaking the (new) tests that use `Promises` 

For some reason unless we remove all calls to `jest.useFakeTimers()` - `return waitFor(() => ...)` does not work